src/hotspot/share/runtime/sharedRuntime.cpp

```diff
@@ -558,6 +558,10 @@ address SharedRuntime::raw_exception_handler_for_return_address(JavaThread* curr
   return NULL;
 }
 
+JRT_LEAF(void, SharedRuntime::coroutine_switch_trace(JavaThread* current, void* arg))
```

Comment:
How about compiling this function only in debug version?

Reply:
If we keep trace_switch and print_site, we can put all these in debug version. Otherwise we can remove them all.

Reply:
We put these functions into debug version.